staging/src/k8s.io/apimachinery/pkg/test/example/types.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package example
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type (
+	ConditionStatus  string
+	PodConditionType string
+	PodPhase         string
+	RestartPolicy    string
+)
+
+// Pod is a collection of containers, used as either input (create, update) or as output (list, get).
+type Pod struct {
```

Comment:
> And why not pkg/apis/example? No reason to have black swarn test packages.

Because its  called a `Pod` and we will go insane talking about "which pod".